src/main/java/appeng/me/GridNode.java

```diff
@@ -603,7 +604,7 @@ public void setControllerRoute( IPathItem fast, boolean zeroOut )
 	{
 		if( zeroOut )
 		{
-			this.channelData &= ~0xff;
+			this.usedChannels &= ~0xffffffff;
```

Comment:
This is actually quite stupid. Previously it just changed the LSB to zero, which was necessary. Now just assigning 0 makes way more sense.